package.json

```diff
@@ -102,7 +102,7 @@
     "html-webpack-plugin": "^5.5.0",
     "identity-obj-proxy": "^3.0.0",
     "istanbul-lib-coverage": "^3.2.0",
-    "jest": "^27.4.7",
+    "jest": "^27.4.1",
```

Comment:
I think I did it because there types aren't updated for 27.4.7
So even though we can technically use the new features, typescript doesn't like it.

The particular feature I found missing was this one:
```
    expect(simp.toJSON()).toMatchObject({
      fixPrefix: false,
      mathjs: "Unit",
      unit: "mi",
      // This is supported in Jest but the types don't allow it
      // value: expect.closeTo(0.00055, 5)
    });
```
I had to work around that by adding `expect(simp.toJSON().value).toBeCloseTo(0.000555, 5);`.

Reply:
So I'm not sure what the right solution is. If we use 27.4.7 it is weird when we can't use features until looking at the version of the types. Or if we leave it at 27.4.1 it isn't obvious why it is out of date. I really wish we could add comments to package.json.

Reply:
Two thoughts:
1. Have you tried `27.5.1`?
2. `dependencies-notes.md`

Reply:
As far as I can tell there are no `@types/jest` for 27.5.1 either.
But I found a bit more about this and might try `@jest/types` instead.